src/config.ts

```diff
@@ -146,4 +146,9 @@ export interface RoutesConfig {
    * Authentication Module for express, hapi and koa
    */
   authenticationModule?: string;
+
+  /**
+   * Controllers glob path, contains directories with wild card to file names.
+   */
+  controllers?: string[];
```

Comment:
Can you rename this to be `controllerPathGlobs` so that it’s clear to consumers?